modules/formatter/src/writer/IdlWriter.scala

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2022 Disney Streaming
+ *
+ * Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package smithytranslate
+package formatter
+package writers
+
+import ast.Idl
+import ControlWriter.controlSectionWriter
+import MetadataWriter.metadataSectionWriter
+import ShapeWriter.shapeSectionWriter
+import WhiteSpaceWriter.wsWriter
+import Writer.WriterOps
+
+object IdlWriter {
+  implicit val idlWriter: Writer[Idl] = Writer.write { idl =>
```

Comment:
can you move that instance inside the `Idl` companion object

Reply:
no all writers should be in the writer package , or they should all be in companion objects, due to the size of some writers ive chosen to move them all into a separate package